### PR TITLE
Integration test setup

### PR DESCRIPTION
**Pull request by quinnjn**
### Summary
Integration test setup for Drop-in web. Heavily references our [restricted-input](https://github.com/braintree/restricted-input) integration test setup.

There is one test that PayPal tokenizes so far. The goal of this PR was to get integration test setup, and we can add more integration tests afterwards.

### Checklist

- [X] Ran integration test

### CODE REVIEW

File: spec/sauce_helper.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+PLATFORM = ENV["PLATFORM"]
+
+def select_browsers
+  browsers = []
+
+  if !PLATFORM || PLATFORM == "desktop"
+    browsers += [
+      ["Windows 10", "chrome", nil],
+      # Firefox >= 48 on Sauce is failing all tests
```
Comment (EvanHahn) [inline spec/sauce_helper.rb]: Is this still the case?

**Comment by quinnjn** [comment]
Pushed a test for tokenizing a card. It does not work on Safari though.

**Comment by lilaconlee** [comment]
Only last thought is to add something to `DEVELOPMENT.md` about how to get things set up

**Comment by quinnjn** [comment]
@lilaconlee I'm not opposed to adding details about the integration test setup in `DEVELOPMENT.md` but ideally integration tests should come for free on open PRs when we add Travis CI. I think this is how we are handling it in restricted-input.

Folks wishing to contribute shouldn't have to worry about trying integration tests until they submit the PR.

**Comment by lilaconlee** [comment]
@quinnjn I was just thinking of something so internal folks know how to run them which could be there or somewhere else

**Pull request merged**